guides/presence.md

```diff
@@ -4,6 +4,8 @@ Phoenix Presence is a feature which allows you to register process information o
 
 Phoenix Presence is special for a number of reasons. It has no single point of failure, no single source of truth, relies entirely on the standard library with no operational dependencies and self heals. This is all handled with a conflict-free replicated data type (CRDT) protocol.
 
+Presence is designed for situations where topic subscription information should be broadcast to connected channels. It is possible to use `Phoenix.Tracker` directly in other use cases, such as wanting to manually dispatch or skip change events.
```

Comment:
I believe the guides are used as entry-point, so I am not sure having this quote at the very beginning will be helpful, because users will likely be unaware of what this all means or even confused by it. My suggestion is to keep this information exclusively in the docs and expand it in there a bit.

Reply:
Yes that is fair. I was bouncing that around in my head